scripts/amm_price_example.py

```diff
@@ -0,0 +1,49 @@
+from hummingbot.core.event.events import TradeType
+from hummingbot.core.gateway.gateway_http_client import GatewayHttpClient
+from hummingbot.core.utils.async_utils import safe_ensure_future
+from hummingbot.strategy.script_strategy_base import Decimal, ScriptStrategyBase
+
+
+class AmmPriceExample(ScriptStrategyBase):
+    """
+    This example shows how to call the /amm/price Gateway endpoint to fetch price for a swap
+    """
+    # swap params
+    connector_chain_network = "uniswap_ethereum_goerli"
+    trading_pair = {"WETH-DAI"}
```

Comment:
it's better to just use the string here so later one you don't have to manipulate the variable to get the trading pair